frontend/pages/404.tsx

```diff
@@ -0,0 +1,44 @@
+import { Center, Box, Text, Button, Container, Flex } from '@chakra-ui/react';
+import { DEFAULT_STORE_CODE } from '@config/env';
+import { DefaultLayout, DefaultLayoutProps } from '@layouts/default';
+import { getGlobalSettings } from '@models/global-settings';
+import { getStoreByCode, getStoreList } from '@models/store';
+
+export default function Custom404({
+   currentStore,
+   stores,
+   globalSettings,
+}: DefaultLayoutProps) {
+   return (
+      <DefaultLayout {...{ currentStore, stores, globalSettings }}>
+         {' '}
```

Comment:
I'm curious what this string is for

Reply:
I think when I commited the files, the linter added that line there? Not super sure though, but I'll be removing it in the next commit